src/Microsoft.VisualStudio.Editors/PropPages/xlf/compileproppage2.de.xlf

```diff
@@ -44,7 +44,7 @@
       </trans-unit>
       <trans-unit id="OptionStrictLabel.Text">
         <source>Option &amp;strict:</source>
-        <target state="translated">Option "&amp;strict":</target>
+        <target state="translated">Option &amp;Strict:</target>
```

Comment:
Why is "Strict" upper case?

Reply:
This change is to make string consistent with other similar ones:
Option In&amp;fer:
Option C&amp;ompare: etc

Reply:
Why is German casing different to the English casing?

Reply:
I don't know if it's relevant here but in German all nouns are capitalized, not just proper nouns.